.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
 * [ ] Have you followed the guidelines in our
-      [Contributing document](https://dvc.org/doc/user-guide/contributing)?
+      [Contributing document](https://dvc.org/doc/user-guide/contributing/code)?
```

Comment:
let's actually rename it code -> core. code is ambiguous to be honest, since we have code for the dvc.org as well. I've renames some titles already in the dvc.org repo to make them explicit.